pkg/meta/base.go

```diff
@@ -277,50 +278,49 @@ func (m *baseMeta) groupBatch(batch map[Ino]dirStat, size int) [][]Ino {
 	return batches
 }
 
-func (m *baseMeta) calcDirStat(ctx Context, ino Ino) (space, inodes uint64, err error) {
+func (m *baseMeta) calcDirStat(ctx Context, ino Ino) (length, space, inodes uint64, err error) {
 	var entries []*Entry
 	if eno := m.en.doReaddir(ctx, ino, 1, &entries, -1); eno != 0 {
 		err = errors.Wrap(eno, "calc dir stat")
 		return
 	}
 	for _, e := range entries {
 		inodes += 1
-		var newSpace uint64
+		var l uint64
 		if e.Attr.Typ == TypeFile {
-			newSpace = e.Attr.Length
+			l = e.Attr.Length
```

Comment:
also add length for symlink

Reply:
After this PR, both of the fast summary and strict summary won't count the length of symlink.